scripts/mkcloud

```diff
@@ -540,12 +540,36 @@ function setupadmin()
     fi
 }
 
+function wait_for_node_shutdown()
+{
+    wait_for 150 1 "virsh domstate $1 | grep shut.off" '$1 node to shut down'
+}
+
+function remove_snapshot_volume()
+{
+    if [[ "${1}" =~ \.snap$ ]]; then
```

Comment:
We could also move the whole naming convention into the function, i.e. we call the `.*snapshot.*` functions with a bare volume name and then append `.snap` to the names as needed.